src/js/components/Chart/doc.js

```diff
@@ -58,8 +58,8 @@ export const doc = Chart => {
     id: PropTypes.string.description(`A unique identifier for the Chart. This
       is required if more than one Chart is shown and they use color
       gradients.`),
-    onClick: PropTypes.func.description(`Called when the user clicks on it.
-      This is only available when the type is line or area.`),
+    onClick: PropTypes.func.description(`Called when the user clicks on the
+     visualization. values[].onClick should be used for types bar and point.`),
```

Comment:
what happens if I use it for line chart?

Reply:
Clicking on the line will trigger the function.

Reply:
So why does it say it should be used for 
>values[].onClick should be used for types bar and point

...and not for a line chart? I feel like I'm missing something.

Reply:
I've revised the documentation for `onClick` in an attempt to make it clearer.